### PR TITLE
Fix and specs for missing plus in login attribute

### PR DESCRIPTION
**Pull request by oliverguenther**
Fixes the merge issue from https://github.com/opf/openproject/commit/105934a5f9da08db69f324632326566b2fd29153

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://oliverguenther-fix-login-missing-plus.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://oliverguenther-fix-login-missing-plus.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -93,6 +93,30 @@
       end
     end
 
+    context 'plus symbol' do
```
Comment (NobodysNightmare) [inline spec/models/user_spec.rb]: you put this context inside `describe 'login whitespace'`...

**Comment by NobodysNightmare** [comment]
Just to maximize the OCD level: What about `.` and `_`? ^^

edit: not really neccessary, I just noticed...

**Comment by oliverguenther** [comment]
Woops...  Mistakes in the tiniest PRs :sob:, will update the symbol sepcs accompanied by the remaining two.

**Comment by NobodysNightmare** [comment]
LGTM :+1: 

Lets wait if travis agrees...

**Comment by linki** [comment]
related to https://community.openproject.org/work_packages/20297 (internal)

**Pull request merged**